data-models/location/gps/schema.json

```
@@ -0,0 +1,23 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/akenza-io/device-type-library/main/data-models/location/gps/schema.json",
+    "title": "GPS",
+    "description": "GPS measurement.",
+    "type": "object",
+    "$defs": {
+        "latitude": {
+            "measurementType": "location/gps/latitude/degrees",
+            "title": "Latitude",
+            "description": "The latitude of a GPS measurement.",
+            "type": "number",
+            "unit": "°"
+        },
+        "longitude": {
+            "measurementType": "location/gps/longitude/degrees",
+            "title": "Longitude",
+            "description": "The longitude of a GPS measurement.",
+            "type": "number",
+            "unit": "°"
+        }
```

```suggestion
        },
        "required": ["latitude", "longitude"]
```

I guess we could add a `required` here, as I suppose we always use both in combination.